package/draft/package

```diff
@@ -43,3 +43,30 @@ package() {
 
     install -D -m 644 -t "$pkgdir"/lib/systemd/system "$srcdir"/draft.service
 }
+
+configure() {
+    systemctl daemon-reload
+
+    if ! is-enabled "$pkgname.service"; then
+        echo ""
+        echo "Run the following command(s) to use $pkgname as your launcher"
+
+        for conflict in $(get-conflicts "$pkgname.service"); do
+            if is-enabled "$conflict"; then
+                echo "$ systemctl disable --now ${conflict/.service/}"
+            fi
+        done
+
+        echo "$ systemctl enable --now $pkgname"
+        echo ""
+    fi
+}
```

Comment:
can this be pushed into some common library or is it fine to duplicate this across launcher's packages?